Dockerfile

```diff
@@ -62,19 +62,18 @@ RUN /mumble/scripts/clone.sh \
     && /mumble/scripts/build.sh \
     && /mumble/scripts/copy_one_of.sh ./scripts/murmur.ini ./auxiliary_files/mumble-server.ini default_config.ini
 
+RUN git clone https://github.com/ncopa/su-exec.git /mumble/repo/su-exec \
```

Comment:
Vulnerable to supply chain attacks if hostile code is introduced in this repository, please change by adding a checkout to a known commit hash after cloning, e.g.:
`git checkout 4c3bb42b093f14da70d8ab924b487ccfbb1397af && \` (current state of master as of 16.1.2025, commit is from 19.12.2022) before cd-ing into the directory and building the software.

Reply:
Please create a PR with the fix

Reply:
For future reference:
When leaving support for e.g. future alpine based containers away, su-exec could be entirely avoided by using:
`setpriv --euid="${PUID}" --egid="${PGID}" --inh-caps=-all -- "${server_invocation[@]}"`

Reply:
Oh, this seems like a native (as in "preinstalled") version of `su-exec`. Is that correct?

And what do you mean with "leaving support for xxx away"? That there are base images which don't come with this command pre-installed?
I think Ubuntu probably does ship it and since that's what we are currently using, I think nothing speaks against switching to using `setpriv` instead. :thinking:

Reply:
setpriv is from the `util-linux` package which is tagged as essential for ubuntu, e.g. focal: https://packages.ubuntu.com/noble/util-linux .
I mean that using this would make it harder to switch to alpine deployment containers in the future (as setpriv from busybox lacks --euid and --egid parameter support).